__tests__/components/JellyForm.spec.js

```diff
@@ -0,0 +1,396 @@
+/* globals expect, describe, it */
+import Promise from 'bluebird'
+import { mount } from 'enzyme'
+import React from 'react'
+import renderer from 'react-test-renderer'
+import sinon from 'sinon'
+import Alert from '../../src/components/Alert'
+import Provider from '../../src/components/Provider'
+
+import JellyForm from '../../src/unstable/components/JellyForm'
+
+const schema = `
+  title: Pokèmon
+  version: 1
+  properties:
+    - Name:
+        type: string
+  additionalProperties: true
```

Comment:
would it work without `additionalProperties` here ?

Reply:
Yeah, it makes no difference here